modules/data-entry/src/main/frontend/src/questionnaire/FormEntry.jsx

```diff
@@ -44,16 +44,30 @@ export const ENTRY_TYPES = QUESTION_TYPES.concat(SECTION_TYPES);
  * @param {string} path the path to the parent of the question
  * @param {Object} existingAnswer form data that may include answers already submitted for this component
  * @param {string} key the node name of the question definition JCR node
+ * @param {Object} classes style classes
  * @returns a React component that renders the question
  */
-let displayQuestion = (questionDefinition, path, existingAnswer, key) => {
+let displayQuestion = (questionDefinition, path, existingAnswer, key, classes) => {
+  const entry = location.hash.substr(1);
+  const anchor = entry ? entry[1] : '';
```

Comment:
This is a string, not an array, and as it is it just ends up being just the second character from the anchor.

```suggestion
  const anchor = location.hash.substr(1);
```